{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings/base.py

```diff
@@ -21,6 +21,8 @@
 
 DEBUG = False
 
+DEMO_SITE = 'DEMO_SITE' in os.environ
```

Comment:
One minor request, move this to the bottom of the file - as this is something a bit custom.